src/frontend/staff-portal/src/app/services/lookups.service.ts

```diff
@@ -97,7 +97,7 @@ export class LookupsService implements ILookupsService {
   }
 
   public getStatuteString(statute: Statute): string {
-    return `${statute.code} ${statute.descriptionText}`;
+    return `${statute.actCode} ${statute.code} ${statute.descriptionText}`;
```

Comment:
may need to remove spaces here

Reply:
Except that Lorraine wrote the logic to split this string on spaces to parse out the actCode and statuteCode. ([here](https://github.com/bcgov/jag-traffic-courts-online/pull/1507/files#diff-f1fc152d71b79e01bbd1dd4c85833393df0523d828479dff6081c296f36d7ddcR704))